k8s/overlays/prod/patches/invoice-cronjob.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: invoice-cronjob
+spec:
+  schedule: "0 13 1 * 0" # TODO Is this the correct schedule?
```

Comment:
You don't need to redefine the `schedule` here, since these are already defined in the base. You only define things that you're changing or adding. Same for `ImagePullPolicy`, `command`, `args`, `restartPolicy`.

Reply:
@joachimweyl Yes. Every comment so far is resolved. I did make a significant addition to the PR, so I would like another review on it.